modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_data.rb

```diff
@@ -99,6 +99,20 @@ def rep_phone_line_number
           higher_level_review.rep_phone_data&.dig('phoneNumber')&.last(4)
         end
 
+        ### The rep signature content not currently printed on form, but kept for if/when we need it
```

Comment:
Not sure if we want to leave this here, but figured I'd let y'all help me decide.

Reply:
I vote on leaving these out. Keeping the bounding boxes around seems OK since that's more meticulous work that we don't need to repeat, but this feels preemptive.

Reply:
cool